Makefile

```diff
@@ -44,7 +44,12 @@ build-requirements:
 update-kinto-admin:
 	rm -fr kinto_admin/static/*
 	npm install -g kinto-admin
-	kinto-admin build -d kinto_admin/static/
+	rm -fr tmp
+	git clone https://github.com/Kinto/kinto-admin.git tmp
+	cd tmp; git checkout v$$(kinto-admin --version); npm install
```

Comment:
hmm not sure the tags are consistently listed in the desired order, sorting by version seems safer.